### ISSUE TITLE
Create a cron job to check this project with the latest OpenThread?

### ISSUE THREAD

**Issue by bukepo** [openthread/ot-rtos#17]
**Is your feature request related to a problem? Please describe.**
It's possible to find this project is not compatible with the latest OpenThread too late. A daily cron job in Travis could help us to find the incompatibility in time.

**Describe the solution you'd like**
Create travis job to check with the latest OpenThread.

**Additional context**
https://github.com/openthread/ot-rtos/pull/16

**Comment by jwhui** [comment]
Doing this would mean that Travis would report the master branch as broken, even when it is not really broken since submodule points to an older commit of openthread/openthread.

Thoughts?

**Comment by bukepo** [comment]
Yes. How about we create a new project which just contains Travis scripts to check compatibility among all OpenThread projects, such as OpenThread, Wpantund, ot-br-posix and ot-rtos?
